src/Data/Digit.purs

```diff
@@ -38,7 +39,7 @@ fromChar c = let code = toCharCode c
                     else Nothing
 
 toChar :: Digit -> Char
-toChar (Digit c) = fromCharCode (c + 48)
+toChar (Digit c) = unsafePartial $ fromJust $ fromCharCode (c + 48)
```

Comment:
This seems pretty bad but I believe it's equivalent to the previous implementation and I wasn't sure how to proceed otherwise.

Reply:
While I'm not 100% sure that this implementation (`unsafePartial <<< fromJust`) will succeed in all cases -- and might therefore cause a runtime exception -- in this case `Digit` is bounded between 0 and 9. So here this will always succeed in this function and I think is worth keeping in this form rather than incur an unnecessary `Maybe` return value.